ddtrace/trace/_limits.py

```diff
@@ -0,0 +1,23 @@
+"""
```

Comment:
Should this be a SpanProcessor? I can see this issue occurring in other integrations

Reply:
At some point, yes. It's not exactly trivial as some values like `elasticsearch.body` need to be obfuscated and can't be truncated as it could result in sensitive data being leaked.

In order to generally truncate, obfuscation needs to be done within the library.